cloud/pkg/cloudhub/servers/server.go

```diff
@@ -30,30 +30,26 @@ func StartCloudHub(messageHandler handler.Handler) {
 func createTLSConfig(ca, cert, key []byte) tls.Config {
 	// init certificate
 	pool := x509.NewCertPool()
-	ok := pool.AppendCertsFromPEM(pem.EncodeToMemory(&pem.Block{Type: certutil.CertificateBlockType, Bytes: ca}))
-	if !ok {
-		panic(fmt.Errorf("fail to load ca content"))
-	}
+    ok := pool.AppendCertsFromPEM(pem.EncodeToMemory(&pem.Block{Type: certutil.CertificateBlockType, Bytes: ca}))
+    if !ok {
+        klog.Exit(fmt.Errorf("fail to load ca content"))
```

Comment:
We can use klog.Exit() to pass errors to top caller, but we need to use klog.OnExit at cmd runner to catch the error.

Exiting directly will destroy the graceful shutdown process.